docs/en/wallet-attestation.rst

```diff
@@ -559,7 +559,25 @@ The details of the revocation mechanism used by the Wallet Provider as well as t
 During the *Wallet Instance initialization and registration* phase the Wallet Provider MAY associate the Wallet Instance with a specific User, subject to obtaining the User's consent. The Wallet Provider MUST evaluate the operating system and general technical capabilities of the device to check compliance with the technical and security requirements and to produce the Wallet Instance metadata.
 When the User consents to being linked with the Wallet Instance, they gain the ability to directly request Wallet revocation from the Wallet Provider, and it also allows the Wallet Provider to revoke the Wallet Instance associated with that User.
 
+The following situations could occur:
 
+- The smartphone is lost;
+- The smartphone has been compromised (e.g., a malicious actor gains control of the smartphone);
+- The smartphone has been reset to factory settings;
+- Other scenarios where the holder can no longer access their Wallet Instance.
+
+If any of these situations occur, the holder **MUST** revoke the Wallet Instance.
+
+.. note::
+
+the *Holder* is defined as the user that has activated and has full control of an instance of the EUDI wallet and the related private keys
```

Comment:
```suggestion
```

we have the section with the defined terms for this. Please do not define terms along the documents sections